_couches/dimipiraat.md

```diff
@@ -2,14 +2,15 @@
 name: Dimi
 city: Cape Town
 country: ZA
-region: Brooklyn
+region: Simon's Town
 email: dimi@zonk.be
+website: https://zonk.be
+mastodon: https://sunbeam.city/@dimi
 github: dimipiraat
```

Comment:
If you've lost control of the @dimipiraat account, perhaps you should also update the `github:` tag in the couch file. :)

Reply:
(Why does GitHub assign this review comment to the wrong line?  I'm sure the requested change is obvious though.)